framer/Material-UI.framerfx/code/CircularProgress.tsx

```diff
@@ -42,12 +40,7 @@ addPropertyControls(CircularProgress, {
     type: ControlType.Number,
     title: 'Value',
     hidden: function hidden(props) {
-      return props.variant === 'indeterminate';
+      return true;
```

Comment:
Is this fine? The default for variant is `'indeterminate'`, that's why I returned `true`

Reply:
I'm not sure, notice how the value widget is gone when the variant is changed:

<img width="246" alt="Capture d’écran 2020-08-13 à 15 27 31" src="https://user-images.githubusercontent.com/3165635/90140096-8a579b80-dd79-11ea-8fa2-d0ae34636d05.png">
<img width="241" alt="Capture d’écran 2020-08-13 à 15 27 38" src="https://user-images.githubusercontent.com/3165635/90140110-8e83b900-dd79-11ea-8f3c-621edd10c976.png">

Reply:
I reverted this as it was before, but now that we do not have variant select, this will always be hidden, maybe it's  better to just show it always (return `true`)?

Reply:
Why do we need to change it? Let's say we solve the variant issue, it seems that we will want to keep the same logic.

Reply:
Added @ts-ignore and left it as it was 👍

Reply:
Locally `yarn workspace framer build` is failing for me:
```
$ cross-env BABEL_ENV=test babel-node --config-file ../babel.config.js ./scripts/buildFramer.js ../packages/material-ui-styles/src ./Material-UI.framerfx/code
$ prettier --write --config ../prettier.config.js './**/*.{js,tsx}'
[error] No files matching the pattern were found: "'./**/*.{js,tsx}'".
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed.
Exit code: 2
Command: C:\Program Files\nodejs\node.exe
Arguments: D:\workspace\material-ui\.yarn\releases\yarn-1.22.4.js build
Directory: D:\workspace\material-ui\framer
Output:

info Visit https://yarnpkg.com/en/docs/cli/workspace for documentation about this command.
```

Any idea how we can fix this?